lib/ar2dto/has_dto.rb

```diff
@@ -31,7 +34,10 @@ def has_dto
     module InstanceMethods
       # @api public
       def to_dto
-        "#{self.class.name}DTO".constantize.new(attributes)
+        class_prefix = AR2DTO::Config.instance.class_prefix
+        class_name = self.class.name.split("::").last
+        class_suffix = AR2DTO::Config.instance.class_suffix
+        "#{class_prefix}#{class_name}#{class_suffix}".constantize.new(attributes)
```

Comment:
Could the dto class name be moved into a method, considering that both the class and the instance method have a similar way to generate it?